src/frontend/screens/Settings/ProjectSettings/DeviceName/EditScreen.tsx

```diff
@@ -147,10 +147,9 @@ export const EditScreen = ({
     [
       handleSubmit,
       navigation,
-      editDeviceInfoMutation.mutate,
-      editDeviceInfoMutation.isPending,
+      editDeviceInfoMutation,
       nameHasChanges,
-      errorModal.openSheet,
```

Comment:
if you extract the field first and use that, it should work e.g.

```ts
const {openSheet} = useBottomSheetModal({openOnMount: false});
```

i think adding the whole object is not desirable (in theory) because the effect will change when one of its fields changes (e.g. `isOpen`). maybe it's not a big issue in practice though 🤷